### PR TITLE
DevEx: eliminate todos - DRY up fakeFile

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/presenter/sequences/navigateToEditOrderSequence.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const navigateToEditOrderSequence = [
   convertHtml2PdfSequence,
   overwriteOrderFileAction,
   {
-    error: [], // TODO: show an alert if this occurs?
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/navigateToEditOrderSequence.js:33]: This issue is on another Trello card:
https://trello.com/c/QgTBG1sU/277-how-should-we-handle-http-errors-from-actions-some-errors-may-currently-be-ignored-silent

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js [hunk 2]
```diff
@@ -49,20 +49,14 @@ export const setDocketEntryMetaFormForEditAction = ({
         document => docketRecordEntry.documentId === document.documentId,
       ) || {};
 
-    // TODO: Abstract this (also in getFormattedCaseDetail)
     if (docketRecordEntry.servedPartiesCode) {
       documentDetail.servedPartiesCode = docketRecordEntry.servedPartiesCode;
+    } else if (documentDetail.servedAt) {
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js]: If both `servedPartiesCode` and `servedAt` have value, `documentDetail.servedPartiesCode` will be set twice, right? Is that intentional?

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.test.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import { applicationContextForClient } from '../../../../../shared/src/business/test/createTestApplicationContext';
+import { applicationContextForClient as applicationContext } from '../../../../../shared/src/business/test/createTestApplicationContext';
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.test.js:1]: 🎉

File: shared/src/business/utilities/getFormattedCaseDetail.js [hunk 4]
```diff
@@ -17,6 +17,21 @@ const courtIssuedDocumentTypes = COURT_ISSUED_EVENT_CODES.map(
   courtIssuedDoc => courtIssuedDoc.documentType,
 );
 
+const getServedPartiesCode = servedParties => {
```
Comment (rachelschneiderman) [inline shared/src/business/utilities/getFormattedCaseDetail.js:20]: Are there tests that cover this somewhere?

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js]: I think the intention is to override `docketRecordEntry.servedPartiesCode`, but I'm not 100% following this code or its intention. I'll try to dig into this some more tomorrow.

File: shared/src/business/utilities/getFormattedCaseDetail.js (hunk 4 above)
Comment (rachaelparris) [reply shared/src/business/utilities/getFormattedCaseDetail.js:20]: It was already covered in tests, I just moved it into its own function. I can move the tests around to more explicitly show that they're testing this function, though.

Comment (rachelschneiderman) [reply shared/src/business/utilities/getFormattedCaseDetail.js:20]: Ah gotcha, all good!

**Review by rachelschneiderman — APPROVED** [review]

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js]: I cleaned this up a little bit so it hopefully makes more sense. Thank you for the feedback!

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js [hunk 5]
```diff
@@ -49,15 +49,11 @@ export const setDocketEntryMetaFormForEditAction = ({
         document => docketRecordEntry.documentId === document.documentId,
       ) || {};
 
-    if (docketRecordEntry.servedPartiesCode) {
-      documentDetail.servedPartiesCode = docketRecordEntry.servedPartiesCode;
-    } else if (documentDetail.servedAt) {
-      documentDetail.servedPartiesCode = applicationContext
+    documentDetail.servedPartiesCode =
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js:52]: Amazing! Thank you!

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**